src/sandstorm/spk.c++

```diff
@@ -889,11 +889,99 @@ private:
+        "      icons = (\n"
+        "        # Various icons to represent the app in various contexts.\n"
+        "        appGrid = (svg = embed \"path/to/128x128.svg\"),\n"
+        "        grain = (svg = embed \"path/to/24x24.svg\"),\n"
+        "        market = (svg = embed \"path/to/150x150.svg\"),\n"
+        "        marketBig = (svg = embed \"path/to/300x300.svg\"),\n"
+        "      ),\n"
+        "      website = \"http://example.com\",\n"
+        "      codeUrl = \"http://example.com\",\n"
+        "      # URL of the app's source code repository, e.g. a GitHub URL.\n"
+        "      # Required if you specify a license requiring redistributing code, but optional otherwise.\n"
+        "\n"
+        "      license = (none = void),\n"
+        "      # The license this package is distributed under.\n"
+        "\n"
+        "      categories = [],\n"
+        "      # A list of categories/genres to which this app belongs, sorted with best fit first.\n"
+        "      # See the list of categories in\n"
+        "      # https://github.com/sandstorm-io/sandstorm/blob/master/src/sandstorm/package.capnp\n"
+        "\n"
+        "      author = (\n"
+        "        # Fields relating to the author of this app.\n"
+        "\n"
+        "        contactEmail = \"youremail@example.com\",\n"
+        "        # Email address to contact for any issues with this app. This includes end-user support\n"
+        "        # requests as well as app store administrator requests, so it is very important that this be a\n"
+        "        # valid address with someone paying attention to it.\n"
+        "\n"
+        "        pgpSignature = embed \"path/to/pgp-signature\",\n"
+        "        # PGP signature attesting responsibility for the app ID. This is a binary-format detached\n"
+        "        # signature of the following ASCII message (not including the quotes, no newlines, and\n"
+        "        # replacing <app-id> with the standard base-32 text format of the app's ID):\n"
+        "        #\n"
+        "        # \"I am the author of the Sandstorm.io app with the following ID: <app-id>\"\n"
+        "        #\n"
+        "        # You can create a signature file using `gpg` like so:\n"
+        "        #\n"
+        "        #     echo -n \"I am the author of the Sandstorm.io app with the following ID: <app-id>\" | gpg --sign > pgp-signature\n"
+        "        #\n"
+        "        # To learn how to set up gpg, visit Keybase (https://keybase.io) -- they have excellent\n"
+        "        # documentation and tools. Moreover, if you create a Keybase account for your key and follow\n"
+        "        # Keybase's instructions to link it to other social accounts (like your Github account), then\n"
+        "        # the Sandstorm app install flow and app market will present this information to the user as\n"
+        "        # \"verified\".\n"
+        "\n"
+        "        #upstreamAuthor = \"Example App Team\",\n"
```

Comment:
Why this one commented out but the packager not commented out?

I think uncomment this one!